dbt_project.yml

```diff
@@ -1,5 +1,8 @@
 name: 'redshift'
-version: '0.0.1'
+version: '0.2.3'
```

Comment:
Extraordinarily minor comment, since `version` serves no functional purpose whatsoever: `0.2.3` is the prior release, this code is going to be part of an `0.3.0` release.